.github/workflows/ci.yaml

```diff
@@ -126,7 +126,7 @@ jobs:
         run: npm run docker
 
       - name: Seed database
-        run: npm run setup
+        run: DATABASE_URL=mysql://starchart:starchart_password@localhost:3306/starchart npm run setup
```

Comment:
I think we should do this instead:

```yaml
run: |
  cp .env.example .env
  npm run setup
```

This way we can make sure that any other environment variables we expect will also be there.

Reply:
Thanks for this! Didn't consider doing this